src/System.IO.FileSystem/tests/File/Move.cs

```diff
@@ -176,22 +176,6 @@ public void MaxPath_Windows()
             });
         }
 
-        [Fact]
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
-        public void LongPath()
-        {
-            //Create a destination path longer than the traditional Windows limit of 256 characters
-            string testFileSource = Path.Combine(TestDirectory, GetTestFileName());
-            File.Create(testFileSource).Dispose();
-
-            Assert.All(IOInputs.GetPathsLongerThanMaxLongPath(GetTestFilePath()), (path) =>
-            {
-                Assert.Throws<PathTooLongException>(() => Move(testFileSource, path));
```

Comment:
For PathHelpers.Windows.cs I only saw it in one place and that change is in here:

![image](https://cloud.githubusercontent.com/assets/450490/20776824/f03b6512-b730-11e6-923f-2013d5358bd4.png)

I made the other changes and I am testing them now.